src/js/letters/containers/LetterList.jsx

```diff
@@ -25,6 +25,8 @@ export class LetterList extends React.Component {
             Inquiry Routing & Information System (IRIS)</a>. VA will respond within 5 business days.
           </p>
         );
+      } else if (letter.letterType === LETTER_TYPES.proofOfService) {
+        letterTitle = 'Proof of Service Card';
```

Comment:
Do we need to define content like we define content in the `else`?

Reply:
oops! good catch, will fix that

Reply:
fixed!